spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/sql/TestCreateTable.java

```diff
@@ -40,8 +40,12 @@
 import org.junit.Test;
 
 public class TestCreateTable extends SparkCatalogTestBase {
+
+  private final boolean isHadoopCatalog;
+
   public TestCreateTable(String catalogName, String implementation, Map<String, String> config) {
     super(catalogName, implementation, config);
+    this.isHadoopCatalog = "testhadoop".equals(catalogName);
```

Comment:
There's no  need for a field. Can you just move this test into the `Assume` line?